### PR TITLE
perf: utils refactor & cleanup

### PR DESCRIPTION
**Pull request by tsi**

### CODE REVIEW

File: src/plugins/playlist/utils/dom.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+const wrap = (el, wrapper) => {
+  el.parentNode.insertBefore(wrapper, el);
+  wrapper.appendChild(el);
```
Comment (ehab-cl) [inline src/plugins/playlist/utils/dom.js:3]: I know that this is an old function: but does not the code eventually leaves duplicate el one that is wrapped and one that is not wrapped? I mean this seems intended but i find it a bit weird, i thought this function would be deleting the "unwrapped element"?

File: src/plugins/playlist/utils/time.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+    timestamp = Date.UTC(struct[1], struct[2], struct[3], struct[4], struct[5] + minutesOffset, struct[6], struct[7]);
+  } else {
+    timestamp = NaN;
+  }
+
+  return timestamp;
+};
+
+export { parseISO8601 };
```
Comment (ehab-cl) [inline src/plugins/playlist/utils/time.js:39]: Oh later we should definitely use a parser form a library 😄

File: src/utils/cloudinary.js [hunk 3]
```diff
@@ -74,38 +73,6 @@ const handleCldError = (that, options) => {
   }
 };
 
-// for IE 11
-function parseUri(str) {
```
Comment (ehab-cl) [inline src/utils/cloudinary.js:78]: 🕺

**Review by ehab-cl — APPROVED** [review]
💪

File: src/plugins/playlist/utils/dom.js (hunk 1 above)
Comment (tsi) [reply src/plugins/playlist/utils/dom.js:3]: No, it inserts `wrapper` before `el` and then moves `el` into `wrapper`

File: src/plugins/playlist/utils/time.js (hunk 2 above)
Comment (tsi) [reply src/plugins/playlist/utils/time.js:39]: Well, yeah, there's so much more we can do... we can go forever

File: src/plugins/playlist/utils/dom.js (hunk 1 above)
Comment (ehab-cl) [reply src/plugins/playlist/utils/dom.js:3]: @tsi but it does not remove el from its parent right?

**Pull request merged**